test/test_gpu2.cpp

```diff
@@ -7296,6 +7296,44 @@ TEST_F(NVFuserTest, FusionPredicateElimination8_CUDA) {
       "T6 should not be predicated");
 }
 
+TEST_F(NVFuserTest, FusionPredicateElimination9_CUDA) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  const int M = 1024, split = 512;
+
+  // Algorithm
+  auto tv0 = makeContigTensor(1, DataType::Float);
+  auto tv1 = set(tv0);
+
+  fusion->addInput(tv0);
+  fusion->addOutput(tv1);
+
+  // Schedule
+  auto tv0c = tv0->cacheAfter();
+  tv0c->setMemoryType(MemoryType::Shared);
+  tv0c->axis(-1)->parallelize(ParallelType::TIDx);
+
+  tv1->split(-1, split);
+  tv1->axis(-1)->parallelize(ParallelType::TIDx);
+
+  // Validation
+  at::manual_seed(0);
+  auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+  auto t0 = at::randn({M}, options);
+
+  {
+    GpuLower gpulw(fusion.get());
+    TORCH_CHECK(PredicatedChecker::isPredicated(tv0c, gpulw));
```

Comment:
nit: Use `EXPECT_FALSE` and `EXPECT_TRUE` instead of `TORCH_CHECK`.

Reply:
Updated checks with `EXPECTED_FALSE`/`EXPECT_TRUE` macros.

Thanks for the suggestion.